src/components/ImageDetails/CopyLicense.vue

```diff
@@ -55,7 +55,7 @@
         </div>
         <div :class="tabClass(1, 'tabs-panel')">
           <textarea id="attribution-html"
-                    class="textarea is-family-monospace is-paddingless"
+                    class="textarea is-family-JetBrains-Mono is-paddingless has-text-weight-normal"
```

Comment:
You should use the `.monospace` class, see the vocabulary docs https://cc-vocabulary.netlify.app/?path=/docs/tokens-typography--figma. Also no need to add this `. has-text-weight-normal` class

Reply:
@brenoferreira so what exactly needs to be changed, I applied the changes that match the text with [code-normal](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=445%3A1053) as given in #962

Reply:
The issue says the code needs to look like the code font in the design library, which uses a monospaced font. This `is-family-JetBrains-Mono` class doesn't exist in Vocabulary. The `.has-text-weight-normal` isn't necessary either. Hope that clears it up for you